### PR TITLE
Webpack vue

### PR DESCRIPTION
**Pull request by lzy2014love**
add vue style webpack loaders

### CODE REVIEW

File: packages/vue/src/hooks/webpack.ts [hunk 1]
```diff
@@ -1,33 +1,117 @@
...
+}
+
 export default async (context: WebpackContext) => {
     const { configurations } = context;
     const webpackConfig = ConfigurationContext.getFrontendConfiguration(
         configurations
     );
     if (webpackConfig) {
-        const isProd = process.env.NODE_ENV === 'production';
+        const appRootDir = process.cwd();
```
Comment (muxiangqiu) [inline packages/vue/src/hooks/webpack.ts:60]: const { configurations, pkg } = context;
const appRootDir = process.cwd();  替换为  const appRootDir = pkg.projectPath;

File: packages/vue/src/hooks/webpack.ts [hunk 2]
```diff
@@ -1,33 +1,117 @@
...
-        const { webpack } = getFrontendMalaguConfig(context.cfg);
-        const sourceMap = !!webpack.devtool;
-        const defaultLoaderOptions = {
-            css: {},
-            scss: {},
-            sass: {},
-            less: {},
-            stylus: {},
-            postcss: {},
+        const isProd = process.env.NODE_ENV === 'production';
```
Comment (muxiangqiu) [inline packages/vue/src/hooks/webpack.ts:62]: 通过 context.dev 判断本地环境构建，还是线上环境构建

**Review by muxiangqiu — APPROVED** [review]

**Comment by muxiangqiu** [comment]
feat(vue): add style webpack loaders

**Comment by lzy2014love** [comment]
feat(vue): add style webpack loaders to support vue project

**Comment by lzy2014love** [comment]
feat(vue): add style webpack loaders to support vue project

**Pull request merged**